test/system/000-setup.bats

```diff
@@ -2,6 +2,10 @@
 
 load 'libs/helpers'
 
+setup() {
+    _setup_containers_store
```

Comment:
I'd call `_setup_containers_store` as a part of the other setup functions since it is a requirement for all the tests to be non-destructible. Also, it is not pretty to call "private" functions from test files (of course, there are some exceptions - e.g., `000-setup.bats`, `000-teardown.bats`).

Reply:
I've just removed the "private"  underscore. The reason it is called in each setup is because the export statement in the last line of _setup_containers_store is needed to be executed in each test for podman to read the variable CONTAINERS_STORAGE_CONF and point it to the correct file. I did as you told, but the test were failing because of that export not taking effect in the rest of the tests, so I just removed the "private" marking for that function.

Reply:
Fair enough.

Reply:
Looking back, making the function a "private" one makes more sense as it is required for proper execution of the tests.